### PR TITLE
:rocket: - Endpoints are ready, changed data model

### PR DESCRIPTION
**Pull request by dmr4eg**
So guys, I wrote a whole openapi specification, which closes all open tickets with endpoints creation. During this one, I discovered some strange solutions in data model that I decided to change. 

Now endpoints are working and swagger is responding.
Screenshot 2024-10-19 at 10 05 05 PM
Screenshot 2024-10-19 at 10 05 13 PM
Screenshot 2024-10-19 at 10 05 20 PM
Screenshot 2024-10-19 at 10 05 28 PM

### CODE REVIEW

**Review by tobias-le — COMMENTED** [review]
.idea do .gitignore pls

**Review by tobias-le — COMMENTED** [review]
necommituj ty vygenerovany classes, pak se to hrozne spatne reviewuje

**Review by tobias-le — CHANGES_REQUESTED** [review]
gitignore na vsechny .DS_Store pls

**Comment by tobias-le** [comment]
Idealni pripad by bylo, kdybychom mohli pomoci mvn clean install vygenerovat open-api tridy. Neni je potreba commitovat do codebase.

**Comment by tobias-le** [comment]
> Idealni pripad by bylo, kdybychom mohli pomoci mvn clean install vygenerovat open-api tridy. Neni je potreba commitovat do codebase.

Target prosim do gitignore, budeme generovat pomoci mvn clean install is all good bro

**Comment by nesqbo** [comment]
mohl by sis prosim pridat vsechny ds_store do gitignore?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Ignore operating system files
+.DS_Store
```
Comment (tobias-le) [inline .gitignore]: **.DS_Store lepsi ne

File: pom.xml [hunk 2]
```diff
@@ -28,6 +28,7 @@
     </scm>
     <properties>
         <java.version>17</java.version>
+        <springdoc.verision>2.5.0</springdoc.verision>
```
Comment (tobias-le) [inline pom.xml]: version*

**Review by tobias-le — APPROVED** [review]
approved

**Pull request merged**